Practicum/Pract. 04/zad2.cpp

```diff
@@ -0,0 +1,202 @@
+#include <iostream>
+#include <fstream>
+
+constexpr char FILE_NAME[] = "PizzaDeliveries.bin";
+const unsigned MAX_NAME = 50;
+const unsigned MAX_TELEPHONE = 20;
+const unsigned MAX_ADDRESS = 100;
+
+enum class PizzaSize
+{
+	UNKNOWN = -1,
+	SMALL,
+	MEDIUM,
+	LARGE,
+	COUNT
```

Comment:
The good practice is the enums to always start from 0 and also all the values should be explicity assigned a number and not implicitly!!